docs/podman-create.1.md

```diff
@@ -455,7 +455,8 @@ Tune the container's pids limit. Set `-1` to have unlimited pids for the contain
 
 **--pod**=""
 
-Run container in an existing pod
+Run container in an existing pod. If the pod does not exist, it will be automatically created.  To make a pod with
```

Comment:
You've '--pod="" '  and the tests just have "--pod".  I know there's a lot of discussion about what should/should not be passed to this option, but if we by chance just leave it as "--pod" and don't accept any options then we should remove the '='.  If we end up accepting a value, we ought to add "name" or some such here.  Same comment for the next run man page.

Reply:
Pod should require a name.

Reply:
@TomSweeneyRedHat i think this is pervasive in podman and a nit on the cli grammer.  I actually didnt add that; it was there. But i think it is common knowledge that on the command line that --pod=foo and --pod foo is the same thing?